corehq/apps/hqwebapp/static/app_manager/less/new_module_modal.less

```diff
@@ -10,7 +10,8 @@
     align-items: center;
     margin-top: -20px;
     margin-bottom: 20px;
-    color: #676b70;
+    // B5 TODO: After migrated, find a better color that close to #676B70
+    color: gray;
```

Comment:
can we use `@cc-text`? or a color based off of that? `gray` is a css built in and unrelated to our theme, even in the bootstrap 3 stylesheet